tests/TestCase/Database/QueryTest.php

```diff
@@ -2207,6 +2207,21 @@ public function testSelectOffset()
     }
 
     /**
+     * Test Pages number.
+     *
+     * @expectedException \InvalidArgumentException
+     * @expectedExceptionMessage Pages should start at 1.
+     * @return void
+     */
+    public function testPageShouldStartAtOne()
+    {
+        $this->loadFixtures('Comments');
+        $query = new Query($this->connection);
+        $result = $query->from('comments')
+                        ->page(0);
```

Comment:
Please no inline indentation.

Reply:
phpcs is happy

Reply:
"happy" doesnt mean it finds all issues :) Some issues are no properly written sniffs for yet.